jcache/embedded/src/main/java/org/infinispan/jcache/embedded/JCache.java

```diff
@@ -371,7 +371,9 @@ public T call() throws Exception {
    @Override
    public <T> Map<K, EntryProcessorResult<T>> invokeAll(
          Set<? extends K> keys, EntryProcessor<K, V, T> entryProcessor, Object... arguments) {
-      checkNotClosed().checkNull(entryProcessor, "entryProcessor").verifyKeys(keys);
```

Comment:
Keeping them fluent was made on purpose to make it nice and clean, since all operations are validation related. There's nothing wrong with them and you should not modify them for the shake of it really :|